browser/ui/views/tabs/brave_tab_context_menu_contents.cc

```diff
@@ -246,3 +267,66 @@ bool BraveTabContextMenuContents::IsValidContextMenu() const {
 void BraveTabContextMenuContents::OnMenuClosed() {
   menu_closed_ = true;
 }
+
+void BraveTabContextMenuContents::NewSplitView() {
+  auto* model = browser_->tab_strip_model();
+  auto* split_view_data = SplitViewBrowserData::FromBrowser(browser_);
+  CHECK(!split_view_data->IsTabTiled(model->GetTabHandleAt(tab_index_)));
+  chrome::AddTabAt(browser_, GURL("chrome://newtab"), tab_index_ + 1,
+                   /*foreground*/ true);
+  split_view_data->TileTabs(
+      std::make_pair(model->GetTabHandleAt(tab_index_),
+                     model->GetTabHandleAt(tab_index_ + 1)));
+}
+
+void BraveTabContextMenuContents::CloseSplitView() {
+  auto* model = browser_->tab_strip_model();
+  auto tab_handle = model->GetTabHandleAt(tab_index_);
+  auto* split_view_data = SplitViewBrowserData::FromBrowser(browser_);
+  auto tile = split_view_data->GetTile(tab_handle);
+  if (!tile) {
+    return;
+  }
+
+  model->CloseWebContentsAt(model->GetIndexOfTab(tile->second), 0);
+}
+
+void BraveTabContextMenuContents::TileSelectedTabs() {
+  auto* model = static_cast<BraveTabStripModel*>(browser_->tab_strip_model());
+  auto indices = model->GetTabIndicesForCommandAt(tab_index_);
```

Comment:
As the tab strip model knows the selected tabs, this could probably live in `browser_commands` too!

Reply:
I found that we shouldn't depend on selected_indices() replacing GetTabIndecesForCommandAt() as the context menu can be shown for non-selected in active tabs. Let me figure out how we can dedupe these and support Tile tab/Break tile from command at the same time.